src/less/global/ds6/global.less

```diff
@@ -5,6 +5,7 @@ body {
     color: @ds6-color-text-default;
     font-family: @ds6-font-family-default;
     font-size: @ds6-font-size-14;
+    -webkit-text-size-adjust: 100%;
```

Comment:
Hmm, are we sure this should be at the global level and not on a per component basis?